code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm

```diff
@@ -71,9 +71,9 @@
 /mob/living/simple_animal/hostile/abnormality/willyouplay/proc/Tie(mob/living/carbon/human/user, work_type)
 	if(janken == 0)
 		SLEEP_CHECK_DEATH(20)
-		say("A draw. Did you think I wouldn�t play scissors?")
+		say("A draw. Did you think I wouldn�t play scissors?")
 		SLEEP_CHECK_DEATH(20)
-		say("I don�t play with folks who don't trust me.")
+		say("I don�t play with folks who don't trust me.")
```

Comment:
This is out of my jurisdiction but due to how big the PR is ill highlight it. I have no clue what the ? is.

Reply:
yeah uh, honesly no clue how that happened. Thats supposed to be the " ' " character

Reply:
Gotcha!